src/javascript/app_2/App/Containers/Layout/header.jsx

```diff
@@ -26,13 +27,14 @@ const Header = ({
     is_logged_in,
     is_mobile,
     is_virtual,
+    location,
     loginid,
     onClickUpgrade,
     toggleAccountsDialog,
 }) => (
     <header className={classNames('header', {
         'header--is-blurred': is_fully_blurred,
-        'header--show'      : !is_loading,
+        'header--show'      : !is_loading || location.pathname !== routes.trade,
```

Comment:
should it be `location.pathname === routes.trade`, since we want to show it only in trade right ?

Reply:
@cakasuma no, we always want to show the header if the route is not `/trade`. And if it **is** `/trade`, then only if it's not loading.